webapp/channels/src/components/searchable_channel_list.tsx

```diff
@@ -530,13 +545,20 @@ export class SearchableChannelList extends React.PureComponent<Props, State> {
             <div
                 id={'hideJoinedPreferenceCheckbox'}
                 onClick={this.handleChecked}
+                onKeyDown={(e) => {
+                    e.stopPropagation();
+                    if (e.key === 'Enter' || e.key === ' ') {
+                        this.handleChecked();
+                    }
+                }}
+                role='checkbox'
+                aria-checked={this.props.rememberHideJoinedChannelsChecked}
+                aria-label={this.props.rememberHideJoinedChannelsChecked ? this.props.intl.formatMessage({id: 'more_channels.hide_joined_checked', defaultMessage: 'Hide joined channels checkbox, checked'}) : this.props.intl.formatMessage({id: 'more_channels.hide_joined_not_checked', defaultMessage: 'Hide joined channels checkbox, not checked'})}
```

Comment:
Could you update this label to be just "Hide joined channel"? Screen readers already read out that it's a checkbox and that it's checked